examples/onprem_demo/README.md

```diff
@@ -6,13 +6,13 @@ This is a demo of a GCP project protected from internet access using [VPC Servic
 - Creates a project that will be protected with VPC SC with a custom network that has only one `10.7.0.0/16` subnet (purely for the sake of simplicity) in a region of your choice
 - Creates a VPN tunnel connecting the two projects (this could conceptually represent a dedicated interconnect as well)
 - Creates a Cloud Router in each project to exchange network routes between the two environments
-- Launches one virtual machine in the VPC SC project: 
+- Launches one virtual machine in the VPC SC project:
     - one Windows instance without an external IP address
-- Launches two virtual machines in the on-prem project: 
-    - one Linux instance acting as a forward proxy for all internet-bound requests; and 
+- Launches two virtual machines in the on-prem project:
+    - one Linux instance acting as a forward proxy for all internet-bound requests; and
```

Comment:
what this "and" means?

Reply:
As I understand the meaning: ` Launches two virtual machines in the on-prem project: one Linux instance ... ; and one Windows ...`